src/index.html

```diff
@@ -370,7 +370,7 @@ <h2 class="title-with-crown">
               class="form__input form__input--textarea"
               name="clientMessage"
               placeholder="Message"
-            >
+            >fix lint please
```

Comment:
"tag-self-close": "never", change it according to your needs

Misha recommended you to change it here https://github.com/Andrii256/layout_miami/blob/develop/.linthtmlrc.json
why didn't you try? any questions?